app/globals.css

```
@@ -159,24 +121,34 @@
     @apply border-border outline-ring/50;
   }
   html {
-    font-size: 120%; /* Increased from default 16px */
+    font-size: 120%;
   }
   body {
     @apply bg-background text-foreground font-sans;
+    position: relative;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)